### PR TITLE
Makefile stub

### PR DESCRIPTION
**Pull request by thom-at-redhat**
Try 2:
1) Makefile stub, now with TODO's (thanks to joden and apagac!)
2) Added JBDS (Eclipse) specific exceptions for .gitignore
3) Support for Id keyword expansion

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#
+#
+# $Id$
+#
+.PHONY: 2to3 coverage docs init install lint test
+
+# TODO: Change from stub to working code
+2to3:
```
Comment (jamesooden) [inline Makefile]: This needs a comment to explain what it does.   I really don't know what 2to3 could mean.

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	echo futurize
```
Comment (jamesooden) [inline Makefile]: Why are we echoing these instead of running them?

**Comment by apagac** [comment]
The commits here will probably need to be squashed.

**Comment by LandoCalrizzian** [comment]
@thom-at-redhat can we close this PR? Doesn't seem like it is still valid?

**Comment by LandoCalrizzian** [comment]
Closing per @thom-at-redhat

**Pull request closed without merging**